Week3/homework/step2-2.js

```diff
@@ -2,19 +2,32 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
+  const len = Math.abs(stopIndex - startIndex);
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = 0; i <= len; i++) {
+    numbers.push(i + startIndex);
+    if (numbers[i] % 3 === 0 && numbers[i] % 5 === 0) {
```

Comment:
The assignment asked to first create an array and then iterate through the newly created array. So I would expect two `for` loops:

```js
for (let i = 0; i <= len; i++) {
  numbers.push(i + startIndex);
}

for (let i = 0; i < numbers.length; i++) {
  ...
}
```